app/views/users/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= render partial: "partials/user_listing", locals: {user: @user}%>
 <%= render partial: "partials/user_details"%>
```

Comment:
- [ ] From your user_details partial, you are using <% @posts.each do |post| %> but for this show page, you should be displaying information or details of recents posts. You should therefore make use of your `def last_three_posts `method you created in the user model

```
# app/views/users/show.html.erb

[...]

<% @user.recent_post.each do |post| %>

<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>

<% end %>

<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>

```

Reply:
well resolved! ✔️